README.md

```diff
@@ -88,6 +88,10 @@ let customer = new Customer(
     )
 )
 await client.createCustomer(customer);
+
+await client.findCustomer("5eb02857-a71e-4ea2-bcf9-57d8885990b");
+
+await client.findAllCustomers({per_page: 2, page: 3});
```

Comment:
Not directly related to this topic, but I'm wondering if instead of adding the doc in here we should just add a link to the "official documentation". It could make it a bit less painful :D

Reply:
yes the README is a bit big and not readable, I agree!